ledger/controllers/wallet.js

```diff
@@ -450,12 +383,8 @@ const write = function (runtime, apiVersion) {
     const grantTotal = result.grantTotal
 
     if (grantIds) { // some grants were redeemed
```

Comment:
we can delete this branch now that there is no grant redemption for legacy anon wallets